bindings/rust/evmc-vm/src/container.rs

```diff
@@ -126,6 +126,8 @@ mod tests {
 
         let host = ::evmc_sys::evmc_host_interface {
             account_exists: None,
+            access_account: None,
```

Comment:
It just occurred to me `account_exists` should be after the accessing one is we consider some lexical ordering?

Reply:
Well, the ordering here is not lexical, but loosely logical. In
```
account_exists
access_account
access_storage
get_storage
...
```

`access_account` & `access_storage` are together between account and storage operations.

Reply:
I would vote for lexical ordering.

Reply:
I'm also not sure what is the point of bringing up "lexical" ordering. For logical ordering, they more belong to the transaction context, not state access (although name indicates otherwise). For me you can put them last after `emit_log`.

Reply:
The other entries seemed to be lexically ordered, in most cases. I don't really care, it was just a remark.

Reply:
I didn't notice that others are not ordered, then I don't care.

Reply:
Still putting them last for historical and weak ABI stability seems a good option. I can do that in the final rebase.